pywr-schema/src/timeseries/mod.rs

```diff
@@ -207,13 +208,13 @@ impl LoadedTimeseriesCollection {
             .ok_or(TimeseriesError::TimeseriesNotFound(name.to_string()))?;
 
         let array: Array2<f64> = df.to_ndarray::<Float64Type>(IndexOrder::default()).unwrap();
-        let name = format!("timeseries.{}_{}", name, scenario);
```

Comment:
Was this "timeseries" prefix required for some reason?

Reply:
only to miminise the risk of the name clashing with an existing non-timeseries parameter